Node.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+# nuxt.js build output
+.nuxt
```

Comment:
I can't see this mentioned in the documentation link provided. Any other resources that explains this?

Reply:
@shiftkey Whops sorry for previous link, I don't know this one will explain that or not 
https://nuxtjs.org/api/configuration-build/#publicpath

But, you can try yourself to run this repo
https://github.com/nuxt-community/starter-template
then it will output its build file into directory named `.nuxt`